_data/packages.yml

```diff
@@ -314,17 +353,18 @@
       name: Jonny Saunders
       github_username: sneakers-the-rat
   eic:
-      name: Isabel Zimmerman
+      name:
       github_username: isabelizimm
   reviewers:
-    - name: Phil Dong
+    - name:
       github_username: phildong
     - name: Iris Young
       github_username: irisdyoung
     - name: Nima Sarajpoor
       github_username: NimaSarajpoor
   archive: '[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10864492.svg)](https://doi.org/10.5281/zenodo.10864492)'
-  version_accepted: 8.4.0
+  version_accepted: 8.4.0 ([repo](https://github.com/caleb531/automata/releases/tag/v8.4.0),
```

Comment:
I don't understand why this has multiple values separated by commas

Reply:
when something looks really odd like this - it's often good to start by checking the review. Here someone did some odd things in the review
<img width="443" alt="Screenshot 2024-11-21 at 8 11 05 PM" src="https://github.com/user-attachments/assets/75998969-daa1-4550-a06b-abcb93935d52">

my guess is our model just didn't know how to handle the additional information provided after the version number. I think we could set up a validator, however, to parse this into two fields in the future, as this version_accepted field is consistently filled out differently across issues! 

it's really hard to parse human-entered free form text :) 

https://github.com/pyOpenSci/software-submission/issues/152

Reply:
I wonder if it can be separated into 2 fields

a. version (e.g., 8.4.0)
b. link to version release (e.g., https://pypi.org/project/automata-lib/8.4.0 )

Ideally the reviewer/EIC would fix the formatting before merge right, nipping such problems at the bud.

Reply:
That said, if you have a field for the PyPI project page (e.g., https://pypi.org/project/automata-lib/), then the link to version release becomes redundant and thus unnecessary. Astropy needs the PyPI project link anyway:

* https://github.com/pyOpenSci/pyopensci.github.io/issues/390

Reply:
cool. let's carry this conversation over to that issue. i want to resolve the structural issues that are breaking things first. adding a pypi item is related to version but it's also an independent thing because not everyone links to pypi for the version submitted or accepted! 

let's get this build working first and then re-open the discussion of adding new fields. if we do that, we could then add a few notes around the format of fields to avoid the issues we just had to troubleshoot!!